descriptor/wpkh.go

```diff
@@ -0,0 +1,181 @@
+package descriptor
+
+import (
+	"encoding/hex"
+	"errors"
+
+	"github.com/btcsuite/btcutil/hdkeychain"
+
+	"github.com/btcsuite/btcd/txscript"
```

Comment:
I am afraid that for elements we should have a way to extends it since elements has more opcodes. For now it's fine tho